### ISSUE TITLE
gliz watermark remover not always working

### ISSUE THREAD

**Issue by Jacob-spearman** [R3nH4s1ssu3s/RESTORED-blookethacks#3]
i tried to use the watermark remover on the gold quest esp and it just removes the first chest identifier and not the gliz watermark.

**Comment by FallGuysFan1234** [comment]
Yea theres also a little box it doesn't remove either

**Comment by R3nH4s1ssu3s** [comment]
ill look into it

**Comment by R3nH4s1ssu3s** [comment]
it seems like the gliz watermark removal doesnt work properly and needs more fixing

**Comment by Jacob-spearman** [comment]
that will not work for me cause I have to use the javascript in my browser bar. and because I am at school so we can not use bookmarklets or injectables

**Comment by R3nH4s1ssu3s** [comment]
exactly, you can just simply put the script inside the browser bar and execute it, you dont have to use a bookmarklet for the blooketui
